x/evm/types/params.go

```diff
@@ -153,6 +147,19 @@ func validateEIPs(i interface{}) error {
 		return fmt.Errorf("invalid EIP slice type: %T", i)
 	}
 
+	uniqueEIPs := make([]int64, 0, len(eips))
+	duplicateEIPs := make([]int64, 0, len(eips))
```

Comment:
Use a map instead to check uniqueness

Reply:
Didn't know about this best practice yet, thanks! 🙏